components/gateway/README.md

```diff
@@ -1 +1,27 @@
 # Gateway
+
+## Overview
+
+Gateway is a component that aggregates Director's and Connector's GraphQL schemas under one service and then sends proper requests to the Director or Connector accordingly.
```

Comment:
It is not actually true. Gateway is a component that exposes a service through which it sends requests to backing services (Director and Connector). It does not aggregate the GraphQL schemas.